cmd/release-controller/info.go

```diff
@@ -658,7 +658,7 @@ class FileServer(handler):
                     outfile.write("Unable to get release tools: %s" % e.output)
 
             except Exception as e:
-                print("Unable to get release tools for %s: %s" % name, e.message)
+                print("Unable to get release tools for %s: %s" % (name, e.message))
```

Comment:
or you can move to:

```python
print('Unable to get release tools for {}: {}'.format(name, e.message))
```

to match the following `log_error` line.

Reply:
six of one, half dozen of the other :grinning: